variables.tf

```diff
@@ -1,9 +1,11 @@
-variable client_id {}
-
-variable client_secret {}
-
+variable client_id { default = "" }
+variable client_secret { default = "" }
```

Comment:
For consistency can we make this multi-line

Reply:
done